### ISSUE TITLE
CI for viewing logs

### ISSUE THREAD

**Issue by nictuku** [yourbase/yourbase#28]
Now that we have an elastic stack #8, we need a command-line that allows people to query yourbase logs.

kubectl is great for reading and searching logs for individual pods or apps.

We need a `yb logs` command or `yb <foo> logs` sub-command that operate on high-level concepts, such as `services`, `branches` and `releases`. Examples:

```bash
# Show recent CI builds for the go-hellohttp-server
yb ci logs -s go-hellohttp-server

# Show logs for the CI build for a specific branch.
yb ci logs -b feature-branch
```

**Comment by nictuku** [comment]
Yeah, take a look at the updated issue description :-). It would be great if you wanted to get started.

I started to work on the web version of this. We could coordinate or I can let you own both bits. Do you have a preference?

**Comment by kbfastcat** [comment]
@nictuku should we allow a search term as above?

**Comment by nictuku** [comment]
@kbfastcat yeah I think it makes sense.
